package.json

```diff
@@ -7,12 +7,11 @@
     "url": "git://github.com/18F/fec-style.git"
   },
   "scripts": {
-    "build": "npm run build-sass && npm run build-js && npm run build-styleguide && npm run copy-css && npm run copy-images",
+    "build": "npm run build-sass && npm run build-js && npm run build-styleguide && npm run copy-images",
     "build-js": "browserify -t hbsfy --debug fec-template/public/init.js > fec-template/public/main.js",
     "build-sass": "node-sass scss/ -o css/",
     "build-styleguide": "cd fec-template/ && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css",
```

Comment:
Still a reference to `copy-css` here, so `npm run build-styleguide` fails. Should that just be removed?